src/clock-weather-card.ts

```diff
@@ -426,7 +429,9 @@ export class ClockWeatherCard extends LitElement {
       date_pattern: config.date_pattern ?? 'D',
       use_browser_time: config.use_browser_time ?? false,
       time_zone: config.time_zone ?? undefined,
-      show_decimal: config.show_decimal ?? false
+      show_decimal: config.show_decimal ?? false,
+      apparent_sensor: config.apparent_sensor,
```

Comment:
`config.apparent_sensor ?? undefined` to make it more explicit

Reply:
done